src/Data/Yaml/Extra.hs

```diff
@@ -1,17 +1,21 @@
+{-# LANGUAGE ScopedTypeVariables #-}
 -- | Wrappers for Yaml functions to workaround
 -- https://github.com/commercialhaskell/stack/issues/2491.
 -- Import Data.Yaml.Extra in place of Data.Yaml to use this workaround.
 -- Beware these functions construct/deconstruct the entire file at once!
 module Data.Yaml.Extra (decodeFileEither, encodeFile, module Data.Yaml) where
 
+import           Control.Exception
```

Comment:
Please double-check this — I think it's OK (this is a straightforward synchronous exception, handled in the IO monad) but I'm not yet fluent with Haskell exceptions.